### PR TITLE
[ID] Clean up `Medals` and `Medals/Legacy_Beatmap_Packs`

### PR DESCRIPTION
**Pull request by ZeroPyrozen**
## Summary

Part of #6509 

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Medals/id.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 Seluruh beatmap perlu diraih dengan mode permainan [osu!](/wiki/Game_mode/osu!) untuk mendapatkan lencana kecuali paket beatmap tersebut menspesifikasikan mode permainan tertentu. Penggunaan [mod pengurang kesulitan](/wiki/Game_modifier#pengurang-kesulitan) atau [mod peningkat kesulitan](/wiki/Game_modifier#peningkat-kesulitan) diperbolehkan.
 
-Namun perlu diperhatikan bahwa konten-konten dari seluruh Video Game Packs, Anime Packs, Internet! Packs, dan Rhythm Game Packs telah berubah drastis sejak [pengenalan paket-paket tersebut pada 2008](https://osu.ppy.sh/community/forums/topics/1853). Untuk daftar paket-paket beatmap ini pada kondisi aslinya, lihat [Paket Beatmap Lama](/wiki/Medals/Legacy_Beatmap_Packs).
+Namun perlu diperhatikan bahwa konten-konten dari seluruh Video Game Packs, Anime Packs, Internet! Packs, dan Rhythm Game Packs telah berubah drastis sejak [pengenalan paket-paket tersebut pada tahun 2008](https://osu.ppy.sh/community/forums/topics/1853). Untuk daftar paket-paket beatmap ini pada kondisi aslinya, lihat [Paket Beatmap Lama](/wiki/Medals/Legacy_Beatmap_Packs).
```
Comment (Nivalyx) [inline wiki/Medals/id.md:11]: `Meskipun demikian, perlu diperhatikan bahwa isi dari seluruh Video Game Pack, Anime Pack, Internet! Pack, dan Rhythm Game Pack yang ada telah banyak berubah sejak paket-paket tersebut [pertama kali diperkenalkan pada tahun 2008](https://osu.ppy.sh/community/forums/topics/1853)`

File: wiki/Medals/id.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ Karena medali Hush-Hush adalah semacam pencapaian rahasia, persyaratan pembuka m
 
 **Medali Beatmap Spotlights** adalah medali yang diberikan ketika pemain menyelesaikan paket beatmap yang dirilis sebagai bagian dari [Monthly Beatmap Spotlights](https://osu.ppy.sh/home/news/2017-03-18-introducing-to-you-spotlights). Agar medali dalam kategori ini dapat terbuka, pemain perlu bermain seluruh beatmap *pada paket mode game apapun* yang dipertandingkan dalam Beatmap Spotlight ranking chart. Penggunaan [mod pengurang kesulitan](/wiki/Game_modifier#pengurang-kesulitan) atau [mod peningkat kesulitan](/wiki/Game_modifier#peningkat-kesulitan) diperbolehkan.
 
-Meskipun Monthly Beatmap Spotlights telah dihentikan sejak April 2018 untuk digantikan oleh [Seasonal Spotlights](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018), seluruh medali yang bersangkutan masih dapat diperoleh hingga saat ini.
+Meskipun Monthly Beatmap Spotlights telah dihentikan sejak bulan April tahun 2018 untuk digantikan oleh [Seasonal Spotlights](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018), seluruh medali yang bersangkutan masih dapat diperoleh hingga saat ini.
```
Comment (Nivalyx) [inline wiki/Medals/id.md:43]: `Meskipun Monthly Beatmap Spotlights telah digantikan oleh [Seasonal Spotlights](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018) sejak bulan April tahun 2018, seluruh medali yang terkait dengan Monthly Beatmap Spotlights masih dapat diperoleh hingga saat ini.`

**Review by Nivalyx — APPROVED** [review]

**Comment by Nivalyx** [comment]
o for some reason the remark check failed to load on this one but i can assure that this pr should be good (and safe) to merge, yeah

image

image

File: wiki/Medals/id.md [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
 Seluruh beatmap perlu diraih dengan mode permainan [osu!](/wiki/Game_mode/osu!) untuk mendapatkan lencana kecuali paket beatmap tersebut menspesifikasikan mode permainan tertentu. Penggunaan [mod pengurang kesulitan](/wiki/Game_modifier#pengurang-kesulitan) atau [mod peningkat kesulitan](/wiki/Game_modifier#peningkat-kesulitan) diperbolehkan.
 
-Namun perlu diperhatikan bahwa konten-konten dari seluruh Video Game Packs, Anime Packs, Internet! Packs, dan Rhythm Game Packs telah berubah drastis sejak [pengenalan paket-paket tersebut pada 2008](https://osu.ppy.sh/community/forums/topics/1853). Untuk daftar paket-paket beatmap ini pada kondisi aslinya, lihat [Paket Beatmap Lama](/wiki/Medals/Legacy_Beatmap_Packs).
+Meskipun demikian, perlu diperhatikan bahwa isi dari seluruh Video Game Pack, Anime Pack, Internet! Pack, dan Rhythm Game Pack yang ada telah banyak berubah sejak paket-paket tersebut [pertama kali diperkenalkan pada tahun 2008](https://osu.ppy.sh/community/forums/topics/1853).
```
Comment (Walavouchey) [inline wiki/Medals/id.md:11]: I believe the second sentence was removed by mistake

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**